conf.d/main

```diff
@@ -5,18 +5,19 @@ DB_NAME=yii2
 DB_USER=yii2
 DB_PASS=$(mcookie)
 
-SRC=/usr/local/src
 WEBROOT=/var/www/yiiframework
 
 # unpack
-cd $SRC
-tar -zxf yii-*.tgz -C .
 mkdir -p $WEBROOT
-mv $SRC/advanced/* $WEBROOT
-rm $SRC/yii-*.tgz
 
 cd $WEBROOT
 
+composer global require "fxp/composer-asset-plugin:^1.2.0"
+composer create-project yiisoft/yii2-app-advanced advanced 2.0.11
+
+mv advanced/* .
+rm -rf advanced
```

Comment:
why not directly run `composer create-project yiisoft/yii2-app-advanced . 2.0.11` ? There is no need to create project in subfolder and then move it.